.github/workflows/build.yml

```diff
@@ -159,6 +159,6 @@ jobs:
 
     - name: Run tests
       run: |
-        npm run build && npm run test
+        cp -r ../precompile/laos-evolution/contracts contracts && npm run build && npm run test
```

Comment:
The approach of not creating a local copy of the contract is beneficial for maintaining a single source of truth. 
However, this means we cannot run tests locally, which could be a limitation. 
One potential solution could be integrating the contract deployment into the `npm run build` process. 
This could help in local testing, though it's worth considering if this approach might be too unconventional or 'hackish'.

Reply:
I think this might be obsolete change. She already merged another PR where `npm run build` copies contract file to compile and then deletes it.

Reply:
the contract copy is already included into the `npm run build` script